charts/cluster-autoscaler/values.yaml

```
@@ -358,8 +358,8 @@ prometheusRule:
   additionalLabels: {}
   # prometheusRule.namespace -- Namespace which Prometheus is running in.
   namespace: monitoring
-  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to `global.evaluation_interval` if not set).
-  interval: null
+  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the default value of `global.evaluation_interval` (which is 1m) if not set).
```

Related to above comment on emitting the interval if not set.
```suggestion
  # prometheusRule.interval -- How often rules in the group are evaluated (falls back to the Prometheus level value of `global.evaluation_interval` if not set).
```